src/components/RecipeView/RecipeView.vue

```diff
@@ -14,9 +14,18 @@
                 </div>
 
                 <div class="meta">
-                    <h2 class="heading">{{ $store.state.recipe.name }}</h2>
+                    <div class="heading">
+                        <h2 class="mb-0">{{ $store.state.recipe.name }}</h2>
+                        <p v-if="$store.state.recipe.citation">
+                            {{
+                                // TRANSLATORS Indicates citation/source of recipe. Ex. "by Grandma Betty"
+                                t('cookbook', 'by')
+                            }}
+                            <span>{{ $store.state.recipe.citation }}</span>
```

Comment:
This is not a good way to handle this wrt i10n. In other languages, it might not be in the pattern _static text_ plus name. So, this might be better delegated to the translators.

Reply:
Strictly speaking, you are right. I made this decision to allow different styles for the word "by" and the source string, which might even become an HTML element later on, like

```
<p>by <a :href="source-uri">Peter Pan</a></p>
```

But it would be better to translate the whole string including the citation variable, then HTML-escaping the citation, string-replacing the citation in the translated string with the html-escaped one (plus any additional styling, linking, etc) and using v-html in a span. Maybe there is a simpler method that I don't know of?

Reply:
How about something like `'by %s%s%s'` with the three parts `<a href="...">`, `$store.state.recipe.citation`, and `</a>`? In the first part, we would only have to insert the `href` value which can be handled manually in this case.

Theoretically, one could fall back to `by %s` and do the concatenation in JS directly.